src/common/transformations/src/transformations/op_conversions/convert_gather_downgrade.cpp

```diff
@@ -48,6 +48,40 @@ pass::ConvertGather8ToGather7::ConvertGather8ToGather7() {
         if (!gather_v8_node)
             return false;
 
+        auto data = gather_v8_node->input_value(0);
+        auto indices_constant =
+            std::dynamic_pointer_cast<ov::op::v0::Constant>(gather_v8_node->input_value(1).get_node_shared_ptr());
+        auto axis_constant =
+            std::dynamic_pointer_cast<ov::op::v0::Constant>(gather_v8_node->input_value(2).get_node_shared_ptr());
+        if (!indices_constant || !axis_constant)
+            return false;
+
+        auto axis = axis_constant->cast_vector<int64_t>();
+        if (axis.size() != 1) {
+            return false;
+        }
+        auto axis_value = axis[0];
+        // normalize `axis` value if it is negative
+        if (axis_value < 0) {
+            axis_value = axis_value + data.get_partial_shape().rank().get_length();
+        }
+        if (data.get_partial_shape().rank().get_length() < axis_value) {
+            return false;
+        }
+        // check `axis` dimension of data tensor is static
+        if (!data.get_partial_shape()[axis_value].is_static()) {
+            return false;
+        }
+        auto axis_dim = data.get_partial_shape()[axis_value].get_length();
+
+        auto indices = indices_constant->cast_vector<int32_t>();
+        // check all the indices are not negative and not out of bound
+        for (size_t i = 0; i < indices.size(); i++) {
+            if (indices[i] < 0 || indices[i] >= axis_dim) {
```

Comment:
it looks like we can normalize indices < 0 
normalized = indices[i] + axis_dim 
if normalized < axis_dim, it's a valid case for conversion

Reply:
Yes, we can, but I don't know why we should do this?
According to [Gather-8 specification](https://docs.openvino.ai/2023.0/openvino_docs_ops_movement_Gather_8.html) Gather-8 should support negative indices. Also the indices can be out of bound, so, I think, that the plugins should support negative values processing if they support Gather-8 as is.
According to [Gather-7 specification](https://docs.openvino.ai/2023.0/openvino_docs_ops_movement_Gather_7.html) Gather-7 expects only non-negative indices (in other words already normalized)
And, finally, in this PR we're getting rid of `GatherNegativeConstIndicesNormalize()` trnasformation, that which mostly created the original problem and confusion.
I suppose to leave it as is and in case of needed we can easily add indices normalization later

Reply:
I would prefer this normalization to be implemented in this PR, it shouldn't affect the complexity of the transformation, just a couple additional lines in this `for` loop. 
The case when all indices of Gather8 after normalization satisfy the condition 0 <= norm_index < axis_dim is not something special and can occur in usual models.

Reply:
added normalization, thanks

Reply:
thanks! resolved